org.osgi.test.junit5/test.bndrun

```diff
@@ -37,22 +37,6 @@
 	org.apache.felix.logback,\
 	slf4j.api
 -runrequires: \
-	bnd.identity;id='${project.artifactId}-test',\
+	bnd.identity;id='${project.artifactId}-tests',\
 	bnd.identity;id='${junit-jupiter-engine.bsn}',\
 	bnd.identity;id='${junit-platform-launcher.bsn}'
--runbundles: \
```

Comment:
I am not sure I like removing the `-runbundles` from the bndrun file. Is that necessary or a good idea?